types/react-table/index.d.ts

```diff
@@ -193,21 +194,25 @@ export interface UseTableInstanceProps<D extends object> {
     plugins: Array<PluginHook<D>>;
     dispatch: TableDispatch;
```

Comment:
I left this here because if you are trying to write hooks it's still there and needed.  It's all well and good not wanting the user to use this from the table instance, but if you are trying to access this from a hook to provide the instance methods, then you still use this.